app/models/content_item_subscriber_list.rb

```diff
@@ -7,7 +9,8 @@ def initialize(content_item)
   end
 
   def subscription_management_url
```

Comment:
Should this be renamed subscription_management_path?

Although I actually wonder if this class is overstepping it's boundaries knowing the path, perhaps it could return the slug and the [code that calling it](https://github.com/alphagov/email-alert-frontend/blob/d00c3784fe5b1f7f937b593b8a84082849f879d8/app/controllers/content_item_signups_controller.rb#L26) should deal with the path. For example: `redirect_to new_subscription_path(topic_id: signup.topic_id)` ?

Also great stuff fixing this, if you weren't aware this should also resolve: https://github.com/alphagov/email-alert-frontend/issues/913

Reply:
Oh I see this change gets removed later in the PR making most of this comment moot.

Reply:
Oh nice, thanks for pointing out the issue - I've updated the PR description with it.